.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+- [Requesting Features](#requesting-features)
+- [Contributing Code](#contributing-code)
+- [Development Setup](#development-setup)
+- [Code Style](#code-style)
+- [Submitting a Pull Request](#submitting-a-pull-request)
+- [Community and Communication](#community-and-communication)
+
+## Getting Started
+
+Before you start contributing to Olympus Gone Wild, please make sure you have:
+
+- A GitHub account;
+- Familiarity with Godot Engine;
+- Read and understood our [Code of Conduct](CODE_OF_CONDUCT.md) and [Project Best Practices](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices);
+
+## Reporting Bugs
+
+If you encounter a bug or unexpected behavior in the game, please check if it has already been reported by searching the [Issues](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/issues). If it hasn't been reported yet, feel free to open a new issue and provide as much detail as possible, including steps to reproduce the issue.
+
+## Requesting Features
+
+Have an idea for a new feature or improvement? We encourage you to discuss it with the community first by opening a [Feature Request](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/issues) issue. This way, you can gather feedback and refine your proposal before investing time in development.
+
+## Contributing Code
+
+### Development Setup
+
+1. Fork the [repository](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/) to your own GitHub account;
+2. Clone your fork to your local machine;
+3. Set up the Godot project using the Godot 4 version;
+4. Create a new branch for your contribution: `git checkout -b 123-my-issue-title`, [check here for branch name convention](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#branch-naming-convention);
+5. Make your changes, test them thoroughly, and ensure they adhere to the project's code style;
+
+### Code Style
+
+Please follow our [Coding Guidelines/Code Styles](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#code-styles) to maintain consistency in the codebase. We recommend using Godot's scripting language best practices and adhering to the GDScript style guide.
+
+### Submitting a Pull Request
+
+1. Ensure your changes are on a feature branch, not the `main` branch;
+2. Commit your changes with clear and concise messages, [check here commit message convections](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#commit-message-convections);
+3. Push your branch to your GitHub fork: `git push origin 123-my-issue-title`, [check here for branch name convention](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#branch-naming-convention);
+4. Create a Pull Request (PR) from your branch to the `main` project repository;
+5. [Check here for pull request best practices](https://github.com/FlamingoFiestaStudio/OlympusGoneWild/wiki/ProjectBestPractices#pull-requests);
```

Comment:
Points 2 and 3 have their warning message after the description. At point 4 it should be the same.

Reply:
Nice caught